src/translations/en.json

```diff
@@ -67,6 +67,11 @@
   "BookingDatesForm.requiredDate": "Oops, make sure your date is correct!",
   "BookingDatesForm.timeSlotsError": "Loading listing availability failed. Please refresh the page.",
   "BookingDatesForm.youWontBeChargedInfo": "You won't be charged yet",
+  "BookingPanel.closedListingButtonText": "Sorry, this listing has been closed.",
+  "BookingPanel.ctaButtonMessage": "Request to book",
+  "BookingPanel.hostedBy": "Hosted by {name}",
+  "BookingPanel.perUnit": "per night",
```

Comment:
@lyyder We have now made bookingTypeUnit changes easier with having translations for all the different unit types:
https://github.com/sharetribe/flex-template-web/commit/082707d01251ee543c596bf8834bdf448dea0510